dvc/repo/__init__.py

```diff
@@ -109,6 +111,8 @@ def __init__(self, root_dir=None):
 
         self._ignore()
 
+        self.plot_templates = PlotTemplates(self.dvc_dir)
```

Comment:
can we avoid initializing it on a global level? we are slowing down the basic flow this way for a lot of commands that don't need to know anything about plots.